src/databricks/sql/client.py

```diff
@@ -482,21 +482,17 @@ def _handle_staging_remove(self, presigned_url: str, headers: dict = None):
             )
 
     def execute(
-        self, operation: str, parameters: Optional[Dict[str, str]] = None
```

Comment:
If we allowed users to do this I think we should continue to support it, but instead of injecting the parameter values in the driver we should convert to the format expected by the backend and pass it on.